src/main/java/uk/gov/hmcts/reform/judicialapi/elinks/scheduler/ElinksApiJobScheduler.java

```diff
@@ -58,8 +58,7 @@ public class ElinksApiJobScheduler {
     DataloadSchedulerJobRepository dataloadSchedulerJobRepository;
 
 
-    public static final String ELINKS_CONTROLLER_BASE_URL =
-            "/refdata/internal/elink";
+    public static final String ELINKS_CONTROLLER_BASE_URL = "/refdata/internal/elink";
 
     @Scheduled(cron = "${elinks.scheduler.cronExpression}")
     @SchedulerLock(name = "lockedTask", lockAtMostFor = "PT5M", lockAtLeastFor = "PT2M")
```

Comment:
Can we make lockAtMostFor = "PT5M", lockAtLeastFor = "PT2M" as configurable value ? Just check on it

Reply:
Thanks @shivajihmcts  made all values configurable hence marking it as resolved for now.